src/SellerLabs/Beakers/Traits/ModelTestTrait.php

```diff
@@ -176,4 +177,31 @@ public function testRelations($property, $type, $other)
                 break;
         }
     }
+
+
+    /**
+     * Tests relationship are connected to correct tables through correct keys.
+     *
+     * @param string $property
+     * @param string $type
+     * @param string $other
+     *
+     * @dataProvider relationsProvider
+     */
+    public function testRelationCanLoad($property, $type, $other)
+    {
+        $model = $this->make();
+
+        switch ($type) {
+            case RelationType::HAS_ONE:
+            case RelationType::BELONGS_TO:
+                $this->assertTrue(is_null($model->$property));
```

Comment:
no, because we are generating an empty model with no relationships. Triggering `$model->$property`  stars a mysql query and if the tables/keys are wrong and exception would be thrown. If the query is valid we expect 0 results meaning instead of an object we are getting null